console.rst

```diff
@@ -61,10 +61,18 @@ method. Then you can optionally define a help message and the
         ;
     }
 
+Registering the Command
+-----------------------
+
+Symfony commands must be registered as services and :doc:`tagged </service_container/tags>`
+with the ``console.command`` tag. If the PHP class of your command extends from
+:class:`Symfony\\Component\\Console\\Command\\Command`, Symfony does this for
+you automatically.
```

Comment:
This doesn't look right to me. Commands do not need to be registered as services in Symfony 2.7 as they are auto-discovered. Additionally, these will not be tagged automatically by Symfony (autoconfiguration was added in 3.3).

Reply:
So, the conditions in 2.7 and 2.8 are: the command must be stored in `Command/` dir, the class name must end in `Command` and they must extend the base `Command` class?

Reply:
Yes, and IIRC correctly we have a section somewhere in the docs that explains these steps.